frontend/TestRun/StructuredEvent.svelte

```diff
@@ -19,22 +22,75 @@
+        if (showSimilars && similars.length > 0 && Object.keys(similarRunsInfo).length === 0) {
+            fetchingIssues = true;
+            try {
+                const response = await fetch("/api/v1/client/sct/similar_runs_info", {
+                    method: "POST",
+                    headers: {
+                        "Content-Type": "application/json",
+                    },
+                    body: JSON.stringify({
+                        run_ids: similars,
+                    }),
+                });
+
+                if (response.ok) {
+                    const data = await response.json();
+                    if (data.status === "ok") {
+                        similarRunsInfo = data.response;
+                    } else {
+                        console.error("Error fetching similar runs info:", data);
+                    }
+                } else {
+                    console.error("Failed to fetch similar runs info:", response.statusText);
+                }
+            } catch (error) {
+                console.error("Error fetching similar runs info:", error);
+                sendMessage("error", "Error fetching similar runs info", "StructuredEvent::toggleSimilars");
+            } finally {
+                fetchingIssues = false;
+            }
+        }
+    };
 
+    function closeSimilarModal() {
+        showSimilars = false;
+    }
+</script>
 
-<div
-    class:d-none={!display || shouldFilter(filterString)}
-    class="mb-2 p-2 shadow rounded bg-white font-monospace"
->
+<div class:d-none={!display || shouldFilter(filterString)} class="mb-2 p-2 shadow rounded bg-white font-monospace">
     <div class="event-header d-flex align-items-start flex-wrap">
         <div class="ms-2 mb-2 bg-dark text-light rounded px-2">{event.eventType}</div>
         <div class="ms-2 mb-2 rounded px-2 severity-{event.severity.toLowerCase()}">{event.severity}</div>
+        <div
+            class="ms-2 mb-2 rounded px-2 bg-light-two d-flex align-items-center"
+            title="Event period type: {event.fields.period_type}"
+        >
+            <Fa icon={faDotCircle} />
+            <span class="ms-2"
+                >{event.fields.period_type}
+                {#if event.fields.duration}
+                    (duration: {event.fields.duration})
+                {/if}</span
+            >
+        </div>
         {#each event.nemesis ?? [] as nemesis}
             <div class="ms-2 mb-2 rounded px-2 status-{nemesis.status.toLowerCase()}">{nemesis.name}</div>
         {/each}
-        <div class="ms-auto mb-2 rounded px-2">
+        <div class="ms-auto mb-2 rounded px-2 d-flex gap-2">
+            {#if (event.severity === "ERROR" || event.severity === "CRITICAL") && similars.length > 0}
+                <button class="btn btn-warning btn-sm d-flex align-items-center gap-2" on:click={toggleSimilars}>
+                    <Fa icon={faExpand} /><span>View {similars.length} Similar Events</span>
```

Comment:
I think `faSearchengin` might look better here.

Reply:
or `faMagic`

Reply:
I decided to go with faLink - as magic wand is not intuitive either. 
If it's a problem we can discuss it and prepare a quick fix.